src/goto-programs/interpreter.cpp

```diff
@@ -380,7 +381,9 @@ void interpretert::execute_other()
   const irep_idt &statement=pc->code.get_statement();
   if(statement==ID_expression)
   {
-    assert(pc->code.operands().size()==1);
+    DATA_INVARIANT(
+      pc->code.operands().size()==1,
+      "expression statement expected to have one operand");
     mp_vectort rhs;
```

Comment:
I would actually prefer to_code_expression(pc->code).expression() instead of the invariant/assertion here